### ISSUE TITLE
Select all markdwon.

### ISSUE THREAD

**Issue by nickolay-kondratyev** [dendronhq/dendron#3887]
## Please select if your request is either something new or an enhancement ##

- [x] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applies to. (Multiple selections are Possible. You can leave blank if you're not sure.) ##

- [ ] Workspace - VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup - Dendron's Lookup Command
- [ ] Views - Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema - Dendron Schemas
- [ ] Pod - Data import from / export to Dendron
- [ ] Publish - External Site Publish
- [x] Markdown - Markdown features, such as syntax support and features

## Is your feature request related to a problem? Please describe ##

Select all markdown without selecting the frontmatter of the notes.

## Describe the solution you'd like ##

A new dendron command that we can assign a shortcut to which would select the markdown contents of the note without selection of the frontmatter.

## Describe alternatives you've considered ##

Using a mouse to select the markdown.

**Comment by v3ss0n** [comment]
Yeah , the front matter getting selected is quite annoying when i do ctrl +A .
